custom/ewsghana/reports/specific_reports/stock_status_report.py

```diff
@@ -251,7 +251,7 @@ def charts(self):
                                  y_axis=Axis(self.chart_y_label, 'd'))
             chart.x_axis_uses_dates = True
             chart.tooltipFormat = True
-            chart.is_rendered_as_email = self.config['is_rendered_as_email']
+            chart.is_rendered_as_email = self.config.get('is_rendered_as_email')
```

Comment:
Should this be
```python
self.config.get('is_rendered_as_email', False)
```
?   I know `None` is falsy, but 'cause it's a boolean variable it seems like it should just be `False`